### DISCUSSION TITLE
No way to download the 2.1 Gb file

### DISCUSSION THREAD

**Discussion by Mignolyx** [w4ffl35/krita_stable_diffusion#65] [General]
I've tried so many times to download the KSD but, only just 100 mb and the connection is out. 10 times i've tried.
Please, have you some torrent or magnet link to try to download it resuming?

**Comment by w4ffl35** [comment]
I recommend waiting for the next release which I am currently working on. I am opening up all source files (including build files) and will be making more frequent updates to the plugin from that point forward.
